bentoml/_internal/io_descriptors/image.py

```diff
@@ -150,9 +144,9 @@ async def from_http_request(self, request: Request) -> ImageType:
         return PIL.Image.open(io.BytesIO(bytes_))
 
     async def to_http_response(self, obj: ImageType) -> Response:
-        if isinstance(obj, np.ndarray):
+        if TypeRef["numpy.typing.NDArray[t.Any]"]("numpy.ndarray").isinstance(obj):
             image = PIL.Image.fromarray(obj, mode=self._pilmode)
-        elif isinstance(obj, PIL.Image.Image):  # type: ignore
+        elif TypeRef["PIL.Image.Image"]("PIL.Image.Image").isinstance(obj):
             image = obj
```

Comment:
example:
no `type: ignore`; lazy import; works both for runtime and type checkers.